test/examples/ucp_hello_world.c

```diff
@@ -240,15 +251,15 @@ static int run_ucx_client(ucp_worker_h ucp_worker)
         if (ucp_test_mode == TEST_MODE_WAIT) {
             /* Polling incoming events*/
             status = ucp_worker_wait(ucp_worker);
-            CHKERR_JUMP(status != UCS_OK, "ucp_worker_wait\n", err_ep);
+            CHKERR_JUMP(status != UCS_OK, "ucp_worker_wait", err_ep);
         } else if (ucp_test_mode == TEST_MODE_EVENTFD) {
             status = test_poll_wait(ucp_worker);
-            CHKERR_JUMP(status != UCS_OK, "test_poll_wait\n", err_ep);
+            CHKERR_JUMP(status != UCS_OK, "test_poll_wait", err_ep);
         }
     }
 
-    msg = malloc(info_tag.length);
-    CHKERR_JUMP(!msg, "allocate memory\n", err_ep);
+    msg = mem_type_allocators[test_mem_type].malloc(info_tag.length);
```

Comment:
we don't need mem_type_allocators in hello world test, because unlike perftest it does not have loadable modules
on other hand better keep hello world as simple as possible
so better wrap stuff with `#if HAVE_CUDA` macros, if needed can define helper function like "memcpy(src,dest,memtype) , memzero(ptr,len,mem_type), malloc, free, .." but no need to use function pointers

Reply:
done